test/e2e/api/admin/feature.e2e.test.js

```diff
@@ -259,7 +259,8 @@ test.serial('creates new feature toggle without type', async t => {
         enabled: false,
         strategies: [{ name: 'default' }],
     });
-    await request.get('/api/admin/features/com.test.noType').expect(res => {
+    await new Promise(r => setTimeout(r, 200));
```

Comment:
Will be good to stop worrying about this!

Reply:
Yes, definitely. So tempted to expand this PR and add the service layer, but then I'll never get done :stuck_out_tongue_closed_eyes: